### PR TITLE
Switch UX to only use CVC instead of CVC/CVV

### PR DESCRIPTION
**Pull request by joyceqin-stripe**
## Summary

Replaced cvv with cvc for amex
## Motivation

When entering a CVC with an Amex card we currently show CVV. We should change this logic so that we show CVC like we do for all other card brands.
https://jira.corp.stripe.com/browse/MOBILESDK-1635
## Testing

Screenshot 2024-09-10 at 1 04 48 PM
Screenshot 2024-09-10 at 1 06 43 PM
Screenshot 2024-09-10 at 1 08 06 PM

### CODE REVIEW

File: StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Elements/TextField/TextFieldElement+Card.swift [hunk 1]
```diff
@@ -168,11 +168,7 @@ extension TextFieldElement {
         let defaultValue: String?
         let cardBrandProvider: () -> (STPCardBrand)
         var label: String {
-            if cardBrandProvider() == .amex {
-                return String.Localized.cvv
-            } else {
-                return String.Localized.cvc
-            }
+            return String.Localized.cvc
```
Comment (wooj-stripe) [inline StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Elements/TextField/TextFieldElement+Card.swift:171]: can be:
```
        let label = String.Localized.cvc
```

File: StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Views/CVCRecollectionView.swift [hunk 2]
```diff
@@ -65,9 +65,7 @@ class CVCRecollectionView: UIView {
         super.init(frame: .zero)
 
         self.titleLabel.isHidden = mode == .detailedWithInput
-        let brand = (self.paymentMethod.card?.brand ?? .unknown) == .amex
-        ? String.Localized.cvv
-        : String.Localized.cvc
+        let brand = String.Localized.cvc
 
         self.titleLabel.text = String(format: String.Localized.cvc_section_title, brand)
```
Comment (wooj-stripe) [inline StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Views/CVCRecollectionView.swift:70]: No need for a variable anymore, can be:
```
self.titleLabel.text = String(format: String.Localized.cvc_section_title, String.Localized.cvc)
```

**Review by porter-stripe — APPROVED** [review]
nice!

File: StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/CVCReconfirmationViewController.swift [hunk 3]
```diff
@@ -23,7 +23,7 @@ class CVCReconfirmationViewController: UIViewController {
         return navBar
     }()
     private lazy var headerLabel: UILabel = {
-        let headerLabelText = cardBrand == .amex ? String.Localized.confirm_your_cvv : String.Localized.confirm_your_cvc
```
Comment (wooj-stripe) [inline StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/CVCReconfirmationViewController.swift:26]: No need for a variable, can be:
```
header.text = String.Localized.confirm_your_cvc
```

File: StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/CVCReconfirmationVerticalViewController.swift [hunk 4]
```diff
@@ -13,7 +13,7 @@ import UIKit
 
 class CVCReconfirmationVerticalViewController: UIViewController {
     private lazy var headerLabel: UILabel = {
-        let headerLabelText = cardBrand == .amex ? String.Localized.confirm_your_cvv : String.Localized.confirm_your_cvc
```
Comment (wooj-stripe) [inline StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/CVCReconfirmationVerticalViewController.swift:16]: no need for variable. can be:
```
header.text = String.Localized.confirm_your_cvc
```

File: StripePaymentsUI/StripePaymentsUI/Source/Helpers/String+Localized.swift [hunk 5]
```diff
@@ -76,12 +76,9 @@ extension String.Localized {
     @_spi(STP) public static var cvc_section_title: String {
         STPLocalizedString(
             "Confirm your %@",
-            "Section title for entering your CVC/CVV. e.g. 'Confirm your CVC' or 'Confirm your CVV'"
+            "Section title for entering your CVC. e.g. 'Confirm your CVC'"
         )
     }
```
Comment (wooj-stripe) [inline StripePaymentsUI/StripePaymentsUI/Source/Helpers/String+Localized.swift:81]: Seems like we can get rid of `cvc_section_title`, and create a new string that doesn't take in a variable.

Comment (porter-stripe) [reply StripePaymentsUI/StripePaymentsUI/Source/Helpers/String+Localized.swift:81]: ++ to leaving as is, otherwise we need to get new strings translated.

**Pull request merged**